packages/components/src/overlays/Popover/src/Popover.module.css

```diff
@@ -0,0 +1,96 @@
+:where(.hop-Popover *) {
+    font-family: var(--font-family);
+    font-size: var(--font-size);
+    font-weight: var(--font-weight);
+    line-height: var(--line-height);
+    color: var(--color);
```

Comment:
I haven't thought of using variables like this, but after looking at it, I guess it makes sense. Good to know!